.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: 💬 Question
-about: If you need help understanding the HumanConnection.
+about: If you need help understanding HumanConnection.
```

Comment:
@ulfgebhardt sorry for nitpicking: You forgot to add a space. @Hardy-Peaceglobal already told be about how to correctly spell it:

Wrong:
"HumanConnection"
"Human-Connection"
"Human connection"
"HC"

Correct:
**"Human Connection"**